_maps/atlantis.json

```diff
@@ -0,0 +1,29 @@
+{
+	"map_name": "NSV Atlantis",
+	"map_link": "Atlantis",
+	"map_path": "map_files/Atlantis",
+	"map_file": [
+		"Atlantis.dmm",
+		"Atlantis2.dmm"
+	],
+	"traits": [
+		{
+			"Up": 1,
+			"Linkage": "Self"
+		},
+		{
+			"Down": -1,
+			"Linkage": "Self"
+		}
+	],
+	"space_ruin_levels": -1,
+	"space_empty_levels": 0,
+	"shuttles": {
+		"cargo": "cargo_hammerhead",
+		"ferry": "ferry_fancy",
+		"arrival": "arrival_atlas",
+		"emergency": "emergency_donut"
+	},
+	"ship_type": "/obj/structure/overmap/nanotrasen/missile_cruiser/starter",
```

Comment:
This is not a missile cruiser, do not make it a missile_cruiser.

Given the size of the map, it is more likely a heavy cruiser.

Reply:
This one? It has the same stats as Tycoon and Gladius. I guess that would work.
![image](https://user-images.githubusercontent.com/71145590/168229266-1ba9a320-c41e-46f9-b4f9-78d652b3a0de.png)

I suppose this ship was selected as a placeholder anyway... Or maybe it was here because Atlas was a used as a basefiles.